sdk/workflow-tengo/src/pframes/util.lib.tengo

```diff
@@ -107,42 +138,33 @@ _SPEC_INDEX_SCHEMA := {
 }
```

Comment:
for `index` parentAxes were defined as array of column names and pfconv resolved them to indices, but here by some mistake it was exact index from the very begining. fixed pfconv to comply, as that simplifies code